test/index.ts

```diff
@@ -99,6 +99,17 @@ test('.set() - invalid key', t => {
 	}, {message: 'Expected `key` to be of type `string` or `object`, got number'});
 });
 
+test('.appendToArray()', t => {
```

Comment:
This needs an assertion for the case where the value is already set to a different type.

I would also appreciate more assertions if you can try to think of some more.